sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1015,6 +1040,7 @@ export interface FeedOptions extends SharedOptions {
     partitionKey?: PartitionKey;
     populateIndexMetrics?: boolean;
     populateQueryMetrics?: boolean;
+    useAllVersionsAndDeleteFeed?: boolean;
```

Comment:
Both of these fields, useIncrementalFeed and useAllVersionsAndDeleteFeed, cannot be true simultaneously, correct? Should we explore the option of using a combined field or perhaps an enum for better representation?

Reply:
Just for clarification, Java and .NET are moving away from the term "Incremental". We are using "LatestVersion" to replace this after engaging the customer. And FFCF is obviously "AllVersionsAndDeletes".

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/2c7c7ad39dec358f7b75e3158d8739afa688ff9f/Microsoft.Azure.Cosmos/src/ChangeFeed/ChangeFeedMode.cs#L13